testing/src/test/java/io/stargate/it/http/docsapi/BaseDocumentApiV2Test.java

```diff
@@ -2840,6 +2858,42 @@ public void deleteNamespace() throws IOException {
         HttpStatus.SC_NOT_FOUND);
   }
 
+  @Test
+  public void createCollection() throws IOException {
+    String tableName = "tb_createTable_" + System.currentTimeMillis();
+
+    RestUtils.post(
+        authToken,
+        hostWithPort + "/v2/namespaces/" + keyspace + "/collections",
+        "{\"name\" : \"" + tableName + "\"}",
```

Comment:
```suggestion
        String.format("%s/v2/namespaces/%s/collections", hostWithPort, keyspace),
        objectMapper.writeValueAsString(payload),
```

Reply:
Agreed not to produce request bodies with the object mapper.